directory_setup_test.go

```diff
@@ -54,6 +64,34 @@ func testDirectorySetup(t *testing.T, context spec.G, it spec.S) {
 			Expect(filepath.Join(workingDir, "public", "packs")).NotTo(BeADirectory())
 			Expect(filepath.Join(workingDir, "tmp", "cache", "assets")).NotTo(BeADirectory())
 		})
+
+		context("with extra asset paths", func() {
+			it.Before(func() {
+				os.Setenv("BP_RAILS_ASSETS_EXTRA_DESTINATION_PATHS", customAssetsPrecompilePaths)
```

Comment:
Can we use: `t.Setenv("BP_RAILS_ASSETS_EXTRA_DESTINATION_PATHS", customAssetsPrecompilePaths)` here? That way we can get rid of the `it.After` below.